.github/workflows/test_pr.yml

```diff
@@ -34,4 +34,24 @@ jobs:
         with:
           args: deploy -P roster-pr --only hosting:project-dashboard-pull-request
         env:
-          FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
\ No newline at end of file
+        FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
+
+
+# Only run the coverage once
+      name: get value percentage and print
+      run: |
+        PERCENTAGE=$(npm test -- --coverage --coverageReporter='text-summary' | grep 'All files' | cut -d '|' -f 5)
+        echo $PERCENTAGE >> $GITHUB_ENV 
```

Comment:
Check the syntax https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable